app/controllers/game_controller.rb

```diff
@@ -1,35 +1,38 @@
 class GameController < ApplicationController
```

Comment:
Should this be GameSessionController?

Reply:
Thanks for pointing this out. Have made the relevant class name changes and accordingly updated the 'routes' file for correct navigation calling.